features/0023-did-exchange/README.md

```diff
@@ -330,15 +334,15 @@ Possible reasons:
 - Unsupported endpoint protocol
 - Invalid Signature
 
-**response_processing_error**
+##### Response Processing Error
 
 - unknown processing error
 
 ## 3. Exchange Complete
 
 The exchange `complete` message is used to confirm the exchange to the _responder_. This message is **required** in the flow, as it marks the exchange complete. The _responder_ may then invoke any protocols desired based on the context expressed via the `pthid` in the DID Exchange protocol.
 
-#### Example
+### Complete Message Example
```

Comment:
Could this section add a treatment of errors and corresponding problem report error codes for exhange-complete message processing errors; e.g., no such exchange at responder? In the weird case where the responder gets an completion with bad data, there is no direction here so far on how to proceed.

Reply:
Note: Not yet added.

Reply:
Added the same error sections to Complete as with the other message type. For now, no reasons added, and only an "unknown processing error".